spec/lib/declare_schema/interactive_primary_key_spec.rb

```diff
@@ -31,8 +37,8 @@ class Foo < ActiveRecord::Base
 
     ### migrate to
 
-    if Rails::VERSION::MAJOR >= 5
-      # rename to custom primary_key
+    if Rails::VERSION::MAJOR >= 5 && !defined?(Mysql2) # TODO TECH-4814 Put this test back for Mysql2
```

Comment:
Should this be based on the `ActiveRecord::Base.connection` type rather than if you can `require 'mysql2'`? The latter is possible if the mysql2 gem is installed at the global level, while the former will always be true to the configuration of the connection.

Reply:
It's just in a test and this part will be replaced soon with TECH-4814 so I think it's ok.